.travis.yml

```diff
@@ -5,6 +5,10 @@ sudo: required
 services:
   - docker
 script:
-  - make DG="docker run -v $(pwd):/var/dgdir slavek/distgen" GOMD2MAN="docker run -v $(pwd):/go/src/github.com/cpuguy83/go-md2man cpuguy83/go-md2man" build
+  - sudo make test-in-container SELECTORS="--distro fedora-26-x86_64.yaml --multispec-selector variant=upstream"
```

Comment:
please remove `sudo`,  in case you have added your user to docker group, you dont need to run it via `sudo`